tsconfig.dist.json

```diff
@@ -8,9 +8,6 @@
 
     "noEmit": true,
 
-    "target": "ES5",
-    "lib": ["ES6", "ES2017.Object", "DOM", "DOM.Iterable"],
-
```

Comment:
**question:** why remove this? my understanding is that we use this to typecheck the built output in ci, and we need the lib definitions to do so

Reply:
Looking at the CI there are issues with this, I might revert it. It seemed redundant when I looked at it but I might have just gotten something wrong... I thought this file inherits/extends from the main one and these are just duplications.

Reply:
oh, no, it's an entirely separate ts file...